.github/workflows/ci.yml

```diff
@@ -56,7 +58,7 @@ jobs:
         run: tox
 
       - name: Run Coverage
-        if: matrix.python-version == '3.12' && matrix.toxenv == 'django42-drflatest'
+        if: matrix.python-version == '3.12' && (matrix.toxenv == 'django42-drflatest' || matrix.toxenv == 'django52-drflatest')
```

Comment:
Not really necessary, but no big deal.

Reply:
I think we should remove it, we don't want to run coverage multiple times, it might become confusing as the numbers could change between different versions of the tests and  might lead to confusion depending on which test run finishes first.  In general, I think we should run coverage on only the latest supported version.